web/views/homepage_view.ex

```diff
@@ -25,4 +24,19 @@ defmodule App.HomepageView do
     tag in String.split(selected_tags, ",")
   end
   def tag_is_selected(_, _), do: false
+
+  @doc """
+    ## nothing_selected
+    ## returns true when tags are empty, or equal to type
+
+    iex> App.HomepageView.nothing_selected("", "all-category")
```

Comment:
@Danwhy if you add: `@doctest HomepageView, import: true` in the `App.HomepageViewTest` module, then you can reference these functions as: `nothing_selected` instead of `App.HomepageView.nothing_selected`

See [here](http://stackoverflow.com/questions/43242418/elixir-how-can-you-use-an-alias-in-doctest/43246122#43246122)